components/posts/Comment.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { View, Text, StyleSheet, Dimensions } from "react-native";
+import { ThemedIconButton } from "@/components/theme/ThemedIconButton"; 
+
+
+const { width, height } = Dimensions.get("window");
+
+export type CommentType = {
```

Comment:
We should add to it the comment / post above it, in order to create a linked list

Reply:
I agree, I just wanted to wait for the firebase to be implemented on the component, because we need to be able to tag users which wasn't possible then.